sdk/storage/storage-blob/src/generated/src/storageClientContext.ts

```diff
@@ -39,7 +39,7 @@ export class StorageClientContext extends coreHttp.ServiceClient {
 
     super(undefined, options);
 
-    this.version = '2020-04-08';
+    this.version = "2020-04-08";
```

Comment:
The swagger path is now `preview/2020-06-12/blob.json` but version number here doesn't change?

Reply:
Manually changed it back to "2020-04-08" to to run tests against the production. Will undo it when STG76 fully is fully rolled out.